### DISCUSSION TITLE
Update problems

### DISCUSSION THREAD

**Discussion by robs-fotoalbum** [LycheeOrg/Lychee#1817] [Q&A] [answered]
Hello,

I ran into problems after migrating from 4.6.1 to 4.7.3 - the error is 200:

Update not applied: `APP_ENV` in `.env` is `production` and `force_migration_in_production` is set to `0`.

Any ides how to resolve this? Thank you :-)

**Comment by ildyria** [comment, marked as answer]
Either edit your `.env` to reflect `APP_ENV` to e.g. `local` or go to `Settings` -> `More` at the bottom -> look for `force_migration_in_production` set it to `1`.

:warning:  Be sure that you are running the latest dependencies.
4.7.3 REQUIRES php 8.1 minimum.
Similarly composer might need to be updated.

**Reply by robs-fotoalbum** [reply]
Thank you for your help :-) problem solved, working again.
